src/packagedef.jl

```diff
@@ -10,6 +10,13 @@ using .CC:
     BasicBlock, CFG,
     compute_basic_blocks, construct_domtree, construct_postdomtree,
     nearest_common_dominator, postdominates
+
+@static if isdefined(CC, :IRShow)
+    using .CC: IRShow
+else
+    using Base: IRShow
+end
```

Comment:
I think we can just use `CC.IRShow` now? `CC.IRShow === Base.IRShow` when `CC === Base.Compiler`.